render/gles2/renderer.c

```diff
@@ -86,32 +99,6 @@ static void gles2_scissor(struct wlr_renderer *wlr_renderer,
 	POP_GLES2_DEBUG;
 }
 
-static void draw_quad(void) {
```

Comment:
I wonder if it's worth it to keep this function and make it take `pos_loc` and `tex_loc`.

Reply:
Eh, probably not